docker-32/install_miniconda.sh

```diff
@@ -0,0 +1,14 @@
+fname=$(ls Miniconda3-*-x86.sh | tail -n 1)
+# this is Ray Donnelly's custom Miniconda, built with our new gcc toolchain
+bash -x $fname -bfp /opt/miniconda
+rm $fname
+/opt/miniconda/bin/conda config --set show_channel_urls True
+# update --all is pulling in replacements from defaults
+# /opt/miniconda/bin/conda update --yes --all
+/opt/miniconda/bin/conda config --add channels c3i_test
+/opt/miniconda/bin/conda install --yes git conda-build curl anaconda-client
+/opt/miniconda/bin/conda install --yes conda-concourse-ci
+/opt/miniconda/bin/conda config --set add_pip_as_python_dependency False
+/opt/miniconda/bin/conda config --set anaconda_upload True
+/opt/miniconda/bin/conda clean -ptiy
```

Comment:
Let's make the canonical location for the root env everywhere on all unix from here on out `/opt/conda`.  That's what I'm trying to standardize on.